workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/AccountsTicketsRestController.java

```diff
@@ -107,6 +116,57 @@ private long _fetchZendeskOrganizationId(String externalReferenceCode)
 		return 0;
 	}
 
+	private ArrayList<Long> _getTicketIds(JSONArray ticketsJSONArray) {
+		ArrayList<Long> ticketIds = new ArrayList<>();
```

Comment:
We generally use the interface (List) unless the specific implementation is needed (ArrayList)